stock_quantity_history_date/i18n/ja_JP.po

```diff
@@ -0,0 +1,40 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_quantity_history_date
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-09-29 07:02+0000\n"
+"PO-Revision-Date: 2021-09-29 07:02+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_quantity_history_date
+#: model:ir.model,name:stock_quantity_history_date.model_date_range
+#: model:ir.model.fields,field_description:stock_quantity_history_date.field_stock_quantity_history__date_range_id
+msgid "Date Range"
+msgstr "日付範囲"
+
+#. module: stock_quantity_history_date
+#: model:ir.actions.act_window,name:stock_quantity_history_date.date_range_action
+#: model:ir.ui.menu,name:stock_quantity_history_date.menu_valuation
+msgid "Date Ranges"
+msgstr "日付範囲"
+
+#. module: stock_quantity_history_date
+#: model:ir.model.fields,field_description:stock_quantity_history_date.field_date_range__inventory_valuate_date
+#: model:ir.model.fields,field_description:stock_quantity_history_date.field_stock_quantity_history__valuate_date
+msgid "Inventory Valuate Date"
+msgstr "在庫評価日"
+
+#. module: stock_quantity_history_date
+#: model:ir.model,name:stock_quantity_history_date.model_stock_quantity_history
+msgid "Stock Quantity History"
+msgstr "在庫記録日"
```

Comment:
```suggestion
msgstr "在庫数量履歴"
```
The translation is strange.  Please fix the translation in Transifex if it comes from the Odoo standard.

Reply:
Sorry, It was my translation.